CREDITS.md

```diff
@@ -76,11 +79,13 @@ Contributors to the Firecracker repository:
 - Christopher Diehl <diehl.chris24@gmail.com>
 - Christos Katsakioris <ckatsak@cslab.ece.ntua.gr>
 - cneira <cneirabustos@gmail.com>
+- Colin Percival <cperciva@freebsd.org>
 - Colin Percival <cperciva@tarsnap.com>
```

Comment:
nit: we can have in the [.mailmap](https://github.com/firecracker-microvm/firecracker/blob/main/.mailmap)?